src/app/(pages)/clients/components/ClientsTable.tsx

```diff
@@ -73,11 +73,16 @@ type ClientsTableProps = { count?: string; page?: string; me?: boolean };
 
 export default async function ClientsTable(props: Readonly<ClientsTableProps>) {
   const { count, page, me } = props;
-  const itemsPerPage = count ? parseInt(count) : 10;
-  const currentPage = page ? parseInt(page) + 1 : 1;
+  let itemsPerPage = 10;
+  let currentPage = 0;
+
+  itemsPerPage = count ? parseInt(count) || itemsPerPage : itemsPerPage;
+  currentPage = page ? parseInt(page) - 1 || currentPage : currentPage;
+
+  const startIndex = currentPage * itemsPerPage + 1;
   const response = me
-    ? await getMeClientsPage(itemsPerPage, currentPage)
```

Comment:
you don't pass the param {me: true} om the function getMeClientsPage

Reply:
Solved with https://github.com/indigo-iam/iam-dashboard/commit/89f3367343ae1108b41990e377ab1588251ffeda